hicommonwealth/commonwealth
# Added frame for prizes

## Link to Issue
Closes: #9963 

## Description of Changes
- added separate frame for contest prizes

## Test Plan
- create farcaster contest and cast it 
- observe prizes frame
- fund contest with our app
- observer prizes frame again

## Deployment Plan

1. n/a

## Other Considerations

- n/a
